tests/test_taar_amowhitelist.py

```diff
@@ -269,13 +270,13 @@
         },
         "summary": {
             "zh-CN":
-                "\\u4f7f\\u7528OSD Lyrics\\u663e\\u793a\\u8c46\\u74e3\\u7535\\u53f0\\u6b4c\\u8bcd"
+                "\u4f7f\u7528OSD Lyrics\u663e\u793a\u8c46\u74e3\u7535\u53f0\u6b4c\u8bcd"
         },
         "tags": [
             "douban",
             "osd-lyrics",
             "osdlyrics",
-            "\\u8c46\\u74e3"
+            "\u8c46\u74e3"
```

Comment:
with a `u"` or a `"...\\u..."` these statements behave the same in python 2 vs 3, and with this they behave differently. if we're sure we want this, please add a comment in this file indicating why it's okay that this behaves differently in python 2 and 3.

Reply:
Good catch, I'll drop this commit because I thought it had been added by 2to3.

Reply:
separately we should fix cases where we use `"...\u..."` instead of `"...\\u.."` and double the backslash (unless its a `u"`)